### PR TITLE
Hack to prevent flash of unstyled content on 404 page.

### PR DESCRIPTION
**Pull request by rebeccacremona**
Visit any page that 404s locally or on staging, for example, https://capstone-static.lil.tools/asdf.

If you watch carefully (force refresh as necessary), you will see the three paragraphs of text currently included in the 404 pages show up first, unstyled, and then the styled page snap into place. 

That's because "slots" are part of the normal "light" DOM rather than the shadow DOM: the browser renders them before taking care of all the fancy stuff that makes this page display correctly.

This PR adds a hack to prevent that behavior: the paragraphs are inserted with `visibility: hidden`, and then, once our web component act is in gear, CSS kicks in and reveals them.

Toggling the `visibility` seems to give a slightly better experience than toggling `display` from `none` to `block`: toggling `display` removes the elements from the layout, which affects heights, which causes the sticky footer to reposition; this is a little smoother, equally smooth/flashy as it is currently.

### CODE REVIEW

File: src/404.html [hunk 1]
```diff
@@ -88,15 +88,15 @@
 		<main id="main" class="l-interiorPage">
 			<header class="u-bg-gray-500 u-col-span-full">
 				<cap-page-header heading="Not Found.">
-					<p class="u-text-white u-text-serif">
+					<p class="u-text-white u-text-serif" style="visibility: hidden">
```
Comment (tinykite) [inline src/404.html:91]: Did you explore adding an additional utility class here, (for example, `class=u-text-white-u-text-serif u-visibility-hidden`) rather than inline style? That would prevent you from needing to resort to using `!important` in the component itself.  

While testing this in an incognito window, I did not visually perceive a difference between a slotted `p` element with this utility class and an inline style declaration.

Comment (rebeccacremona) [reply src/404.html:91]: Hmm. I wasn't having any luck with a utility class getting picked up in time. Let me try again, maybe I had a typo.

Comment (tinykite) [reply src/404.html:91]: If for whatever reason I was not perceiving the difference between adding a utility class to one paragraph, and leaving in the inline style for other paragraphs (and watching for a change to affect the one paragraph and not others), that's okay — it's super small!

Comment (rebeccacremona) [reply src/404.html:91]: Yeah, I don't think that works:
- added `u-visibility-hidden` to the `<p>`s
- added to utilities.css:
```
.u-visibility-hidden {
	visibility: hidden;
}
```
- switched cap-page-header.js to:
```
::slotted(p) {
    visibility: visible;
```

And, the pargraphs are visible on load (presumably while css is loading), and are hidden after load (presumably because the slotted p selector is less specific than the utility class).

Do you have a different strategy in mind than the above? Did I do that wrong?

Comment (tinykite) [reply src/404.html:91]: I don't think you did this wrong, at all! 

It's a strange situation, and yes — you are absolutely right that class selectors have a slightly greater specificity than element selectors. I'm sure there may be an approach here that would allow us to avoid the !important but I think using it once for an odd edge case like this is absolutely fine.

**Review by tinykite — APPROVED** [review]
